metricbeat/module/rabbitmq/shovel/_meta/fields.yml

```diff
@@ -0,0 +1,30 @@
+- name: shovel
+  type: group
+  release: beta
+  description: >
+    shovel
+  fields:
+    - name: name
+      type: keyword
+      description: >
+        The name of the shovel with non-ASCII characters escaped as in C.
+    - name: vhost
+      type: alias
+      path: rabbitmq.vhost
+      migration: true
+      description: >
+        Virtual host name with non-ASCII characters escaped as in C.
+    - name: node
+      type: alias
+      path: rabbitmq.node.name
+      migration: true
+      description: >
+        Node name.
+    - name: state
```

Comment:
If this field refers to status property in the shovel document then can you rename the field name to status ? and the description should start as Current state of shovel...